src/azure/storage/sas/account_sas.rs

```diff
@@ -0,0 +1,317 @@
+use crate::azure::storage::sas::{urlencoded, Protocol};
+use crate::time::DateTime;
+use crate::{hash, time};
+use bitflags::bitflags;
+use std::fmt;
+
+bitflags! {
+    /// Specifies the signed services that are accessible with the account SAS
+    #[derive(Copy, Clone, PartialEq, Eq)]
+    pub struct AccountSasResource: u8 {
```

Comment:
Too much for now, please handwriting directly. We don't want to support all of them so far.

Reply:
@Xuanwo Honesty is indeed a bit complicated, but if we directly hardcode it, how should we customize the parameters of the token? Or do we not need to support customization at all?

Reply:
> Or do we not need to support customization at all?

Not now. Make it working first and add features while users really need them,

Reply:
okay, will change it later.

Reply:
get it, please take a look @Xuanwo